docs/people/setup.md

```diff
@@ -0,0 +1,26 @@
+---
+id: setup
+title: Setup Developer Account
+sidebar_label: Setup Developer Account
+---
+
+In order to create Apps on Deskera platform, developer needs to register with us as a Partner. 
+
+We uses OAuth 2.0 for the Authentication scheme. Developers must first register with us by providing some details about their app, such as email, callbankURL, and name.
+
+Please send us an email at `care@deskera.com` with the information mentioned below to register as a partner and obtain Deskera `client_id` and `client_secret`
```

Comment:
Got this form now to get teh details https://docs.google.com/forms/d/e/1FAIpQLScZpPZg0Gu18AFF4E7YrekqSWLLbN7MVi6fBxTluQ_KFm4-zw/viewform?usp=sf_link

Reply:
made the necessary changes